platform/viewer/src/components/ViewportGrid.jsx

```diff
@@ -33,81 +36,106 @@ function ViewerViewportGrid(props) {
   } = servicesManager.services;
 
   const updateDisplaysetForViewports = useCallback(
-    displaySets => {
+    async displaySets => {
       const [
         matchDetails,
         hpAlreadyApplied,
       ] = HangingProtocolService.getState();
-      const numViewports = viewportGrid.numRows * viewportGrid.numCols;
-      const seriesNumberParam = getQueryParam('series_number');
+      // const seriesNumberParam = getQueryParam('series_number');
 
-      if (!isParamViewLoaded && seriesNumberParam) {
-        const initialDisplaySet = displaySets.find(ds => {
-          return ds.SeriesNumber === seriesNumberParam;
-        });
+      // if (!isParamViewLoaded && seriesNumberParam) {
+      //   const initialDisplaySet = displaySets.find(ds => {
+      //     return ds.SeriesNumber === seriesNumberParam;
+      //   });
 
-        initialDisplaySet &&
-          viewportGridService.setDisplaysetForViewport({
-            viewportIndex: 0,
-            displaySetInstanceUID: initialDisplaySet.displaySetInstanceUID,
-          });
+      //   initialDisplaySet &&
+      //     viewportGridService.setDisplaysetForViewport({
+      //       viewportIndex: 0,
+      //       displaySetInstanceUID: initialDisplaySet.displaySetInstanceUID,
+      //     });
 
-        initialDisplaySet && setIsParamViewLoaded(true);
-      } else {
-        if (!matchDetails.length) return;
+      //   initialDisplaySet && setIsParamViewLoaded(true);
+      // } else {
```

Comment:
explanation for this too?

Reply:
not working with series number query paramter